docs/docs/faq.md

```diff
@@ -187,7 +181,7 @@ You *don't* need to trust:
 * The hardware manufacturers *other* than the CPU vendor. RAM, disk, firmware, the operating system, PCI devices, and
 other hardware components are all untrusted in the SGX threat model.
 * The owner of the hardware on which your enclave is running. You can verify the remote attestation to ensure that the
-remote system is up to date and that the enclave is running securely.
+remote system is up-to-date and that the enclave is running securely.
```

Comment:
I've see other instances of "up to date" in dcap.md

Reply:
I've changed it back to 'up to date'. IntelliJ marked this as an error and made me think it was incorrect. Here, it is used as an adverb and should not be hyphenated. Thank you. :)